externs/shaka/text.js

```diff
@@ -316,6 +316,20 @@ shaka.extern.Cue = class {
      * @exportDoc
      */
     this.id;
+
+    /**
+     * Nested cues
+     * @type {Array.<!shaka.extern.Cue>}
+     * @exportDoc
+     */
+    this.nestedCues;
+
+    /**
+     * Whether or not the cue only acts as a spacer between two cues
```

Comment:
Can you elaborate on that? How does a cue act as a spacer?

Reply:
That was a way to implement the <br /> inside the TTML bodies, while not making a specific cue/implementation for the TTML (could be a better architecture of doing so though)

A span is appended w/ a display:block between two other cues.

Reply:
Oh, so it's an empty span that does nothing except separate two other spans with context?

Reply:
That's right :)